lambdas/delete-user-data/src/delete-data.ts

```diff
@@ -7,7 +7,7 @@ import { VCItemKey } from "./types";
 const ddbDocClient = DynamoDBDocument.from(
   config.isLocalDev
     ? new DynamoDBClient({ endpoint: config.localDynamoDbEndpoint })
-    : new DynamoDBClient({ region: "eu-west-2" })
+    : new DynamoDBClient({ region: "eu-west-2" }),
```

Comment:
This default has changed in Prettier 3, and is definitely a matter of taste: https://prettier.io/docs/en/options.html#trailing-commas

We could also revert back to the original value with a `.prettierrc`